lib/ar_query_matchers/queries/field_name.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ArQueryMatchers
+  module Queries
+    # An instance of this class is one of the values that could be returned from the QueryFilter#filter_map.
+    # its accepts a name of an ActiveRecord model, for example: 'Company'.
```

Comment:
extremely minor typo
```suggestion
    # it accepts a name of an ActiveRecord model, for example: 'Company'.
```